estuary/utils/general.py

```diff
@@ -399,3 +434,39 @@ def set_story_labels(requested_node_label, results, reverse=False):
         idx += delta
         node_idx += delta
     return results
+
+
+def get_siblings_description(story_node_id, story_node_story_flow, backward):
+    """
+    Generate a description of the siblings.
+
+    :param string story_node_id: the id of the current story node
```

Comment:
"the id of the current story node" => "the id of the current story node that the siblings are related to"